articles/quickstart/backend/_includes/_api_using.md

```diff
@@ -75,6 +75,17 @@ The API will return a 401 HTTP (Unauthorized) status code:
 
 ![Response for unauthorized API request](/media/articles/server-apis/using/private-unauthorized.png)
 
+To test that your API is properly secured, you can obtain a test token in the Auth0 Dashboard:
```

Comment:
The "Obtaining an Access Token" does not describe how to obtain a token with/without scopes to test that the API is configured properly.

I chose to put these instructions in the "Test Your API" section because it seemed specific to the task at hand. 

I think this scenario - _I just followed the tutorial and created APIs secured with and without scopes, now how do I test it is working correctly?_ is not unique to me. I noticed that the [Laravel API Quickstart](https://auth0.com/docs/quickstart/backend/laravel) includes information on using test tokens with/without scopes in the first part of the Quickstart.

I'll try reorganizing the information to obtain test tokens into the "Obtaining an Access Token" section, and see if that flow would make sense. We can follow-up and discuss further tomorrow.

Reply:
After additional conversation, we decided to not include information about obtaining a test token at this time (beyond what is already documented in `_api_using`. This is something to further consider when looking at the overall structure of all API Quickstarts.